soda/scientific/soda/scientific/distribution/comparison.py

```diff
@@ -59,6 +63,12 @@ def __init__(
         dist_name: Union[str, None],
         data: List[Any],
     ):
+        if len(data) == 0:
```

Comment:
@tituskx not sure if we have tests for exceptions but I would add a test to cover this scenerio